web/src/portico/help.js

```diff
@@ -76,10 +76,10 @@ function add_copy_to_clipboard_element($codehilite) {
     });
 }
 
-function render_code_sections() {
-    $(".code-section").each(function () {
+function render_tabbed_section() {
```

Comment:
Since this does iterate over potentially multiple tabbed sections, I'd keep the plural here: `render_tabbed_sections()`